### PR TITLE
chore: add samples code

### PR DESCRIPTION
**Pull request by DominicKramer**
The samples from

https://github.com/GoogleCloudPlatform/nodejs-docs-samples/tree/master/debugger

have been migrated to this library.

Fixes: #577

### CODE REVIEW

**Comment by ofrobots** [comment]
What the status here?

File: samples/README.md [hunk 1]
```diff
@@ -0,0 +1,70 @@
...
+* [Running locally](#running-locally)
+* [Deploying to App Engine](#deploying-to-app-engine)
+* [Running the tests](#running-the-tests)
+
+## Setup
+
+Before you can run or deploy the sample, you need to do the following (where
+appropriate, replace `YOUR_PROJECT_ID` with the ID of your Cloud project):
+
+1.  Refer to the [appengine/README.md][readme] file for instructions on
```
[the remark this answers is not in the document]
Comment (DominicKramer) [reply samples/README.md]: Fixed.

**Comment by DominicKramer** [comment]
This is ready to review now.

File: samples/README.md [hunk 2]
```diff
@@ -12,7 +12,7 @@ This sample demonstrates [StackDriver Debugger][debugger] with Node.js.
 Before you can run or deploy the sample, you need to do the following (where
 appropriate, replace `YOUR_PROJECT_ID` with the ID of your Cloud project):
 
-1.  Refer to the [appengine/README.md][readme] file for instructions on
+1.  Refer to the [App Engine Quickstart][quickstart] for instructions on
```
Comment (kjin) [inline samples/README.md]: Nit: The link here goes to an example deployment, not instructions on deployment. IMO the original link was fine, it just needed to be updated now that we're in a separate repo.

Comment (DominicKramer) [reply samples/README.md]: Thanks.  I've updated the link.

**Pull request merged**